__tests__/steps/markJourney.spec.ts

```diff
@@ -45,5 +47,15 @@ describe('markStep', () => {
         'mark.start_navigate_to_second_screen_first_journey',
       );
     });
+    it('shouldnt mark anything if performance is not supported', () => { 
```

Comment:
What happen to the space between like 49 and 50?
When you code try to always keep tings neat, even with tests.